.travis.yml

```diff
@@ -14,8 +14,11 @@ notifications:
     - henderson.molly4@gmail.com
```

Comment:
hmm might be time to update this. Not really sure when @mollyhenderson has last been in here.